### PR TITLE
robbie_week_2_check

### PR DESCRIPTION
**Pull request by Robbie-Smith**

### CODE REVIEW

File: module-2/backend/week_two.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+   * ActiverRecord is a wrapper(Object Relational Model which associated ORMs with Ruby Objects) for postgresql, that makes it easier for us to work with databases.
```
Comment (s-espinosa) [inline module-2/backend/week_two.md]: 👍

File: module-2/backend/week_two.md [hunk 2]
```diff
@@ -13,8 +14,14 @@ end
 ```
 
 What are some methods you can call on `Team`? If these methods aren't defined in the class, how do you have access to them?
+  * Methods: id, name, owner_id, all, find, find_by, where, create, destroy, group, order
```
Comment (s-espinosa) [inline module-2/backend/week_two.md]: 👍

File: module-2/backend/week_two.md [hunk 3]
```diff
@@ -25,13 +32,27 @@ end
 ```
 
 Now how would you find the owner of the team with an id of 4?
+* owner = Team.find(4).owner
```
Comment (s-espinosa) [inline module-2/backend/week_two.md]: 🎉

File: module-2/backend/week_two.md [hunk 4]
```diff
@@ -25,13 +32,27 @@ end
...
 
-3. What do they allow you to do?
-7. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
-8. Define foreign key, primary key, and schema.
-9. Describe the relationship between a foreign key on one table and a primary key on another table.
-10. What are the parts of an HTTP response?
-11. Describe some techniques to make our Sinatra code more DRY. Give an example of when you would use these techniques.
+5. What do they allow you to do?
```
Comment (s-espinosa) [inline module-2/backend/week_two.md]: Blerg. This is stale. We should have removed it when we updated the questions.

File: module-2/backend/week_two.md [hunk 5]
```diff
@@ -25,13 +32,27 @@ end
...
+  * A student has_many :teachers
+  * A teacher has_many :students
+7. Define foreign key, primary key, and schema.
+  * Primary key is the table being queried
+  * Foreign key is the key that is being compared to the primary key in the table being queried.
+  * The schema shows our tables,rows,columns, values(string, text, integer, float, datetime...)
+8. Describe the relationship between a foreign key on one table and a primary key on another table.
+  * The foreign key on one table is compared to the primary key on the other to determine if the two things are connected.
+9. What are the parts of an HTTP response?
+  * I think the header is another word for the response. The header contains the verb(post,gets...), the response code(200, 302, 404..etc), the redirect(if there is one) it contains the content type(html, character encoding), and it contains the content (which is in different parts of the header depending upon the verb, for example in a post it is in the body/content-length).
```
Comment (s-espinosa) [inline module-2/backend/week_two.md]: Careful to tease out the request from the response here. Request will include your verb, response will include the status code. If you think about it, you send a GET request, or a POST request, etc., and then the server responds with something like `200` meaning that everything's o.k., or `404`, I couldn't find what you were looking for, or `503`, there's been a server error, etc.

Comment (Robbie-Smith) [inline module-2/backend/week_two.md]: Cool thank you.

**Pull request closed without merging**